test/e2e/specs/fakerp/admin.go

```diff
@@ -92,6 +92,11 @@ var _ = Describe("Openshift on Azure admin e2e tests [AzureClusterReader][Fake]"
 		Expect(err).ToNot(HaveOccurred())
 	})
 
+	It("should start node-exporter correctly", func() {
+		err := wait.Poll(2*time.Second, 20*time.Minute, ready.DaemonSetIsReady(cli.AppsV1.DaemonSets("openshift-monitoring"), "node-exporter"))
```

Comment:
Unrelated to PR but I didn't realize we are waiting 20 minutes for a ds deployment.  That seems high.